examples/Dockerfile.centos8

```diff
@@ -18,3 +21,23 @@ RUN    echo "/usr/local/lib" > /etc/ld.so.conf.d/usrlocal.conf \
     && echo "/usr/local/lib64" >> /etc/ld.so.conf.d/usrlocal.conf \
     && ldconfig
 
+# Install packages needed to build el8 rpms.
+RUN yum -y install \
+           autoconf \
+           automake \
+           gcc \
+           make \
+           python3 \
+           python3-sphinx \
+           python3-sphinx_rtd_theme \
+           rpm-build \
+           rpmlint \
+           rsync \
+           wget
+# CentOS8 doesn't have a bats package, thus we install from source
+RUN wget https://github.com/sstephenson/bats/archive/v0.4.0.tar.gz \
+    && tar xf v0.4.0.tar.gz \
+    && bats-0.4.0/install.sh /usr
```

Comment:
Agreed, however this kludge was an attempt to get rpmbuild to think it could satisfy the `/usr/bin/bats` dependency. It didn't work, thus should be removed.